notebooks/DM-31046-Zooniverse-cutouts.ipynb

```diff
@@ -0,0 +1,700 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# DM-31046 Zooniverse cutouts"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "This notebook is to try out the `ZoonverseCutoutsTask` implemented on DM-31046. The notebook may evolve as that task is improved (i.e. to include source selection via sql query) on a subsequent ticket."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import glob\n",
+    "import os\n",
+    "import pandas as pd\n",
+    "import sqlite3\n",
+    "\n",
+    "import IPython.display\n",
+    "\n",
+    "import lsst.daf.persistence\n",
```

Comment:
We're trying to discourage people from using Gen 2 (see numerous support threads on Community). Would it be possible to rewrite this demo using `daf.butler`?

Reply:
I'd be inclined to say that we shouldn't be writing any gen2 tutorials or examples at this point. Gen2 is now formally deprecated (as of v22.0 of pipelines). Gen2 doesn't work on the IDF. This notebook will have to be rewritten before the end of the year to use gen3.

Reply:
The code itself doesn't care about gen2 vs. gen3. I need a dataset to use to make some plots with, and I had access to this dataset, which happened to be gen2.

Reply:
I don't think that's true, actually. `generate_images` takes a Butler as an parameter... surely that's not Gen 2???

Reply:
This is not, nor can it be a `PipelineTask` (no way to define Quanta, given how it uses apdb). The way it uses the `butler` underneath is independent of gen2/gen3.

The final user-facing API is not yet defined, so whether it actually has a `run` in the end is unclear.